extensions/elytron-security-oauth2/runtime/src/main/java/io/quarkus/elytron/security/oauth2/runtime/auth/Oauth2Credential.java

```diff
@@ -1,8 +1,8 @@
 package io.quarkus.elytron.security.oauth2.runtime.auth;
 
-import io.undertow.security.idm.Credential;
+import io.quarkus.elytron.security.runtime.UndertowTokenCredential;
 
-public class Oauth2Credential implements Credential {
+public class Oauth2Credential implements UndertowTokenCredential {
```

Comment:
Just noticed that in both cases the package name is linked to Undertow. Will it work on too of VertX (with all the latest HTTP related changes) ? Will I open an issue to follow up on this one ?